### PR TITLE
Add versioning

### PR DESCRIPTION
**Pull request by rrhan0**
Added version argument to dockerfiles, added manual workflow for choosing a version

### CODE REVIEW

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -7,6 +7,20 @@ on:
     branches: [ "master" ]
   merge_group:
     branches: [ "master" ]
+  workflow_dispatch:
+    inputs:
+      copter:
+        description: Copter Version Number [Releases](https://github.com/ArduPilot/ardupilot/tags)
```
Comment (ElioDiNino) [inline .github/workflows/docker.yml]:
```suggestion
        description: Copter Version (#.#.#): [Releases](https://github.com/ArduPilot/ardupilot/tags)
```

File: .github/workflows/docker.yml [hunk 2]
```diff
@@ -7,6 +7,20 @@ on:
...
+        required: true
+        type: string
+      plane:
+        description: 'Plane Version Number'
```
Comment (ElioDiNino) [inline .github/workflows/docker.yml]:
```suggestion
        description: 'Plane Version (#.#.#): [Releases](https://github.com/ArduPilot/ardupilot/tags)'
```

File: .github/workflows/docker.yml [hunk 3]
```diff
@@ -35,7 +49,8 @@ jobs:
         context: x86/
         platforms: linux/amd64
         push: ${{ github.event_name != 'pull_request' }}
-        tags: ubcuas/uasitl:copter
+        tags: ubcuas/uasitl:copter,ubcuas/uasitl:copter-${{ inputs.copter || env.COPTER_VERSION }}
```
Comment (ElioDiNino) [inline .github/workflows/docker.yml]:
```suggestion
        tags: ubcuas/uasitl:copter-${{ inputs.copter || env.COPTER_VERSION }}
```

**Review by ElioDiNino — CHANGES_REQUESTED** [review]
Looks great overall. Just a few small changes. We should also update the README to reflect that versions are required both for building and pulling the images.

**Review by ElioDiNino — DISMISSED** [review]

File: README.md
Comment (rrhan0) [inline README.md:1]: Does the custom Ardupilot repository still need to be supported?

Comment (ElioDiNino) [reply README.md:1]: Yes, it should be. I don't think any changes are required for this though.

**Review by ElioDiNino — APPROVED** [review]

**Pull request merged**